fastestimator/network.py

```diff
@@ -181,3 +174,62 @@ def _reduce_loss(self, batch, global_batch_size, epoch_losses, warm_up):
                     "please make sure loss is element-wise loss"
             reduced_loss[loss_name] = tf.reduce_sum(element_wise_loss) / global_batch_size
         return reduced_loss
+
+
+def build(model_def, model_name, optimizer, loss_name):
```

Comment:
if we're getting rid of loss_name="loss" (which I'm still not sold on) then shouldn't we also change all of the loss files to not have that default output name?

Reply:
if we are getting rid of default loss_name, then yes!